biggraphite/cli/command_copy.py

```diff
@@ -119,14 +119,24 @@ def _copy_metric(accessor, src_metric, dst_metric, time_start, time_end):
         """Copy a metric."""
         log.info("Copying points from '%s' to '%s'" % (src_metric.name, dst_metric.name))
         for stage in src_metric.retention.stages:
+            aggregated_stage = stage.aggregated()
             rounded_time_start = stage.round_up(time_start)
             rounded_time_end = stage.round_up(time_end)
 
             points = []
-            res = accessor.fetch_points(src_metric, rounded_time_start, rounded_time_end, stage)
-            # TODO (t.chataigner) : store aggregated points correctly
-            for timestamp, value in res:
-                if timestamp >= rounded_time_start and timestamp <= rounded_time_end:
-                    points.append((timestamp, value, 1, stage))
+            query_results = accessor.fetch_points(
+                src_metric, rounded_time_start, rounded_time_end, stage, raw=True)
+            for successful, rows_or_exception in query_results:
+                if not successful:
+                    continue
+                for row in rows_or_exception:
+                    if aggregated_stage:
+                        (time_start_ms, offset, _, value, count) = row
```

Comment:
Hmm.. thinking more about it, you can't just ignore the replica id here ! What you are writing will be completely wrong if aggregation method is 'sum'.